pytest-embedded-jtag/tests/test_jtag.py

```diff
@@ -17,7 +17,7 @@ def test_pexpect_by_jtag(testdir):
         def test_pexpect_by_jtag(dut):
             dut.gdb.gdb_set('mi-async', 'on')
             dut.gdb.file_exec_and_symbols(os.path.join(dut.app.app_path, 'build', 'hello-world.elf'))
-            dut.gdb.interpreter_exec_console(f'source {os.path.join(dut.app.app_path, "gdbinit")}')
+            dut.gdb.interpreter_exec_console(f'source {os.path.join(dut.app.app_path, "prefix_map_gdbinit")}')
```

Comment:
gdbinit is the init commands to interact with openocd server

prefix_map_gdbinit is used when enable reproducible build.

Reply:
In this case  `IdfDut` instance doesn't have an `gdb` attribute. Should we add one more attribute to `IdfDut` ?

Reply:
here it's using the `JtagDut` as the dut class

Reply:
Oh. Sorry. My bad, seems I've changed this line by mistake without noticing it!